aptos-move/framework/aptos-framework/sources/object.move

```diff
@@ -210,23 +209,24 @@ module aptos_framework::object {
         create_object_internal(creator_address, obj_addr, false)
     }
 
-    /// Create a new object from a GUID generated by an account.
-    public fun create_object_from_account(creator: &signer): ConstructorRef {
-        let guid = account::create_guid(creator);
-        create_object_from_guid(signer::address_of(creator), guid)
+    /// Create a new object by generating a random UUID based on transaction hash.
+    /// The created object is deletable as we can guarantee the same UUID can 
+    /// never be regenerated with future txs.
+    public fun create_object(creator: &signer): ConstructorRef {
+        let uuid = aptos_framework::transaction_context::create_uuid();
+        create_object_internal(signer::address_of(creator), uuid, true)
     }
 
-    /// Create a new object from a GUID generated by an object.
-    public fun create_object_from_object(creator: &signer): ConstructorRef acquires ObjectCore {
-        let guid = create_guid(creator);
-        create_object_from_guid(signer::address_of(creator), guid)
+    /// Deprectated. 
+    /// Create a new object by generating a random UUID based on transaction hash.
+    public fun create_object_from_account(creator: &signer): ConstructorRef {
+        create_object(creator)
     }
 
-    fun create_object_from_guid(creator_address: address, guid: guid::GUID): ConstructorRef {
```

Comment:
Can you update the comments in `ConstructorRef` after deleting this? They still refer to this function.

Reply:
Done.